qubesbuilder/plugins/build_windows/__init__.py

```diff
@@ -0,0 +1,534 @@
+# The Qubes OS Project, http://www.qubes-os.org
+#
+# Copyright (C) 2021 Frédéric Pierret (fepitre) <frederic@invisiblethingslab.com>
+# Copyright (C) 2024 Rafał Wojdyła <omeg@invisiblethingslab.com>
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>.
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+import logging
+import os.path
+import re
+import shutil
+import subprocess
+import yaml
+from enum import StrEnum
+from pathlib import Path
+from typing import Dict, ItemsView, List
+
+from qubesadmin import Qubes
+from qubesadmin.exc import QubesException
+from qubesbuilder.component import QubesComponent
+from qubesbuilder.config import Config
+from qubesbuilder.distribution import QubesDistribution
+from qubesbuilder.executors import ExecutorError
+from qubesbuilder.executors.windows import WindowsExecutor
```

Comment:
And similarly this (and the `qubesadmin` imports above) can be problematic. It wants either some wrapping here (and raise an exception early in `run` if qubesadmin is not available), or changing `PluginManager` so that plugin import error is not fatal. IMO the first option is better if wouldn't clutter the code too much.

Reply:
FWIW several CI tests that are unrelated to QWT fail due to importing `qubesadmin`. That could be a good check if it works as expected - tests not related to QWT should pass even if `qubesadmin` cannot be imported.